src/bosquet/openai.clj

```diff
@@ -45,18 +37,18 @@
   be passed to `complete-chat`"
   ([prompt] (complete prompt nil))
   ([prompt {:keys [impl api-key
+                   api-endpoint
                    model temperature max-tokens n top-p
                    presence-penalty frequence-penalty]
             :or   {impl              :openai
-                   api-key           (get-api-key)
                    model             ada
                    temperature       0.2
                    max-tokens        250
                    presence-penalty  0.4
                    frequence-penalty 0.2
                    top-p             1
                    n                 1}}]
-   (let [params {:impl              impl
```

Comment:
due to a change in the [wkok](https://github.com/wkok) the "impl" needs to be given differently.
This is root cause of #9